parameters.py

```diff
@@ -23,7 +23,7 @@
 class ENV:
     REAL: int = 0
     SIMULATION: int = 1
-    status: int = SIMULATION
+    status: int = REAL
```

Comment:
Soon i create and a configuration file that is stored locally only and gitignored to overcome that problem